### ISSUE TITLE
Autobackup doesn't work. Windows 7 64.

### ISSUE THREAD

**Issue by zb13y** [mypaint/mypaint#372]
```python
Mypaint version: 1.2.0-beta.0+gitexport.adc9031
System information: Windows-7-6.1.7601-SP1
Traceback (most recent call last):
  File "C:\Program Files\MyPaint\share\mypaint/gui/autorecover.py", line 144, _recovery_tree_selection_changed_cb(self=<gui.autorecover.Presenter object>, sel=<TreeSelection object at 0xf821550 (GtkTreeSelection at 0xf89f900)>)
            path = model.get_value(iter, self._LISTSTORE_PATH_COLUMN)
            autosave = lib.document.AutosaveInfo.new_for_path(path)
            sensitive = not autosave.cache_in_use
  variables: {'path': ('local', 'C:\\Users\\\xd0\xa0\xd1\x92\xd0\xa0\xc2\xb7\xd0\xa0\xc2\xb0\xd0\xa0\xd1\x98\xd0\xa0\xc2\xb0\xd0\xa1\xe2\x80\x9a\\AppData\\Local\\Microsoft\\Windows\\Temporary Internet Files\\mypaint\\doc.bjcf5h\\autosave'), 'lib.document.AutosaveInfo.new_for_path': ('global', <bound method type.new_for_path of <class 'lib.document.AutosaveInfo'>>), 'autosave': (None, [])}
  File "C:\Program Files\MyPaint\share\mypaint/lib/document.py", line 119, new_for_path(cls=<class 'lib.document.AutosaveInfo'>, path='C:\\Users\\\xd0\xa0\xd1\x92\xd0\xa0\xc2\xb7\xd0\xa0\xc2\xb0\xd0\xa0\xd1\x98\xd0\xa0\xc2\xb0\xd0\xa1\xe2\x80\x9a\\AppData\\Local...porary Internet Files\\mypaint\\doc.bjcf5h\\autosave')
            if not os.path.isdir(path):
                raise ValueError("Autosave folder %r does not exist", path)
            has_data = os.path.isdir(os.path.join(path, "data"))
  variables: {'path': ('local', 'C:\\Users\\\xd0\xa0\xd1\x92\xd0\xa0\xc2\xb7\xd0\xa0\xc2\xb0\xd0\xa0\xd1\x98\xd0\xa0\xc2\xb0\xd0\xa1\xe2\x80\x9a\\AppData\\Local\\Microsoft\\Windows\\Temporary Internet Files\\mypaint\\doc.bjcf5h\\autosave'), 'ValueError': ('builtin', <type 'exceptions.ValueError'>)}
ValueError: ('Autosave folder %r does not exist', 'C:\\Users\\\xd0\xa0\xd1\x92\xd0\xa0\xc2\xb7\xd0\xa0\xc2\xb0\xd0\xa0\xd1\x98\xd0\xa0\xc2\xb0\xd0\xa1\xe2\x80\x9a\\AppData\\Local\\Microsoft\\Windows\\Temporary Internet Files\\mypaint\\doc.bjcf5h\\autosave')
```

I don't have this - Temporary Internet Files\\mypaint\\doc.bjcf5h\\autosave. And idk how to change path if it possible, so autobackup doesn't work.

\xd0\xa0\xd1\x92\xd0\xa0\xc2\xb7\xd0\xa0\xc2\xb0\xd0\xa0\xd1\x98\xd0\xa0\xc2\xb0\xd0\xa1\xe2\x80\x9a - Cyrillic

Sorry for my english.

**Comment by achadwick** [comment]
Non-ASCII usernames seem to be confusing this bit of code. Can it open and save files normally? (first, turn off autosave in the Edit→Prefs window)

**Comment by zb13y** [comment]
I can open, save even i can name file with cyrillic symbols and then open it normally. I can open scrap, path is the same with the cyrillic username. All is working. Only backup doesn't work.

**Comment by achadwick** [comment]
Hi - I think this one should be fixed with the update (v1.2.0-beta.0.73) just released at <https://github.com/mypaint/mypaint/releases/tag/v1.2.0-beta.0>. Could you download it and check please?
